sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -8,9 +8,6 @@
 
 #pragma once
 
-// Suppress a compiler warning about undefined CL_TARGET_OPENCL_VERSION
-// Khronos ICD supports only latest OpenCL version
-#define CL_TARGET_OPENCL_VERSION 220
 #include <CL/cl.h>
```

Comment:
If I understand correctly with this change compiler will emit errors in user's code instead.
Right?

Reply:
Warnings.
AFAIK, we should not expose OpenCL API to user headers. Once transition to PI is completed, user will never see these warnings.

Reply:
I don't think it's a good idea to merge this change until transition to PI is done.
Also there is OpenCL interoperability API, which exposes OpenCL API to user headers.